web-extensions/chrome/scripts/load-bookmarks.js

```diff
@@ -16,12 +16,9 @@ const bmcDb = new BmcDataAPI();
 
 function BmcMangaList() {
     this._node = document.getElementById('manga-list');
-    this._mode = BmcMangaList.MODE_BROWSE;
 }
 
-BmcMangaList.prototype.MODE_REGISTER = 'register';
-BmcMangaList.prototype.MODE_BROWSE = 'browse';
-
+// TODO: should be handled somehow
```

Comment:
so, this is still WIP I guess?

Reply:
BTW, the mode was there to differentiate "alias on click" and "unroll sources on click" in the sidebar.

Reply:
See #51, which will reinstate the feature.